apps/console/src/features/sms-providers/pages/twilio-sms-provider.tsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { FinalFormField, TextFieldAdapter } from "@wso2is/form";
+import {
+    EmphasizedSegment,
+    Hint,
+    PrimaryButton
+} from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement } from "react";
+import { useTranslation } from "react-i18next";
+import { Divider, Grid } from "semantic-ui-react";
+import { SMSProviderConstants } from "../constants";
+
+interface TwilioSMSProviderPageInterface extends IdentifiableComponentInterface {
+    isReadOnly: boolean;
+    onSubmit: (values: any) => void;
+}
+
+const TwilioSMSProvider: FunctionComponent<TwilioSMSProviderPageInterface> = (
+    props: TwilioSMSProviderPageInterface
+): ReactElement => {
+    const {
+        ["data-componentid"]: componentId,
+        onSubmit,
+        isReadOnly
+    } = props;
+    const { t } = useTranslation();
+
+    return (
+        <div>
```

Comment:
Let's use react fragment instead of `<div>` if you don't pass any props to it. Check in other places as well
```suggestion
        <>
```